presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -646,7 +646,11 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
   } // task's pipelines loop
 
   processOperatorStats(prestoTaskStats, taskStatus.state);
-  processTaskStats(prestoTaskStats, taskRuntimeStats, taskStatus.state);
+  processTaskStats(
+      prestoTaskStats,
+      taskRuntimeStats,
+      taskStatus.state,
+      /*tryToSkipIfRunning=*/false);
```

Comment:
Why not changing the underlaying config parameter? skipRuntimeStatsInRunningTaskInfo?
"skip-runtime-stats-in-running-task-info"

Reply:
Good point. Close this.